news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 27 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-07-29 15:30:00
+authors: [ AntoineThomas ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 1st to Sunday 07th of July 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week was a very warm week, indeed. Did you find a solution to stay productive in your area?
+
+If week 26 was a small week if we look at the number of pull request, the week 27 is one of the biggest of this year. Work on 1.7.7 has started on the develop branch, and the work to finish the 1.7.6 has been merged. And, also, a lot of work has been done on modules. Well done!
```

Comment:
```suggestion
If week 26 was a small week considering the numbers of pull requests, week 27 is one of the biggest of this year ! Work on 1.7.7 has started on the develop branch, and the work to finish the 1.7.6 has been merged. Also, a lot of work has been done on modules. Well done!
```

Actually work on 1.7.7 started way before 😅at the feature freeze it started

Reply:
@matks => `of this year!`

Reply:
done directly in the doc